license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/GitPropertiesProvider.java

```diff
@@ -34,6 +33,7 @@
   private static final String COPYRIGHT_LAST_YEAR_MAX_COMMITS_LOOKUP_KEY = "license.git.copyrightLastYearMaxCommitsLookup";
   public static final String COPYRIGHT_LAST_YEAR_SOURCE_KEY = "license.git.copyrightLastYearSource";
   public static final String COPYRIGHT_LAST_YEAR_TIME_ZONE_KEY = "license.git.copyrightLastYearTimeZone";
+  public static final String COMMITS_TO_IGNORE = "license.git.commitsToIgnore";
```

Comment:
This name should include `_KEY` in the name to distinguish it from the value.

Reply:
Roger, thanks for pointing that out